UnitTests/expected/Storyboards-osx-Message-Defaults.swift.out

```diff
@@ -46,22 +46,22 @@ struct StoryboardScene {
 
     case MessageDetailsScene = "MessageDetails"
     static func instantiateMessageDetails() -> NSViewController {
-      return StoryboardScene.Message_Osx.MessageDetailsScene.controller()
+      return StoryboardScene.Message_Osx.MessageDetailsScene.controller() as! NSViewController
```

Comment:
@tbaranes: Could you try in a terminal on your Mac to run the following command and tell me if it finds any warning, especially the one about the trailing newline?

```sh
cat UnitTests/expected/Storyboards-All-Default.swift.out | swiftlint lint --use-stdin
```

If so, maybe that means that your `cat` appends some newline at the end of the output after printing the content of the file, while mine doesn't? (maybe you use a different shell? I tried with `zsh` and `sh` both but couldn't reproduce the warning)